crates/shared/src/order_validation.rs

```diff
@@ -387,6 +390,7 @@ impl OrderValidating for OrderValidator {
                     convert_signing_scheme_into_quote_signing_scheme(
                         order.signature.scheme(),
                         true,
+                        additional_gas,
                     )?,
                 )
                 .await?,
```

Comment:
Can't add comment to relevant line 👇, but I realized that:
```rust
```

Should likely be:
```rust
        let full_fee_amount = quote
            .as_ref()
            .map(|quote| quote.data.fee_parameters.unsubsidized_with_additional_cost(additional_gas))
            .unwrap_or_default();
```